README.md

```diff
@@ -37,9 +39,13 @@ This file contains paths to find supporting files, meta data, and dummy data to
 name: Figure Image
 description: A `figure` element with `figcaption` and an included `img` element from /base/img/img.html.
 twig: ./figure-image.twig
-sass: ./sass/_figure-image.scss
+sass: ./_figure-image.scss
 script: ./figure-image.js
 category: components
+subcategory: media
+atomic:
```

Comment:
We can. I guess I wanted to give the option to use atomic or another categorization option as well. That way, we can satisfy multiple system. Atomic is *very specific* to patternlab and it's own nomenclature. I was hoping we could have categories *in addition* to the atomic structure.

If it's too confusing though, we can just have category/subcat - probably a lot easier